contact-duplicate/src/app/extensions/package.json

```diff
@@ -6,5 +6,15 @@
   "dependencies": {
     "@hubspot/ui-extensions": "latest",
     "react": "^18.2.0"
+  },
+  "scripts": {
+    "lint": "eslint . --ext .tsx,.ts",
+    "lint:fix": "eslint . --ext .tsx,.ts --fix"
+  },
+  "devDependencies": {
+    "@hubspot/eslint-config-ui-extensions": "^0.7.2",
```

Comment:
@brandenrodgers for project templates, how should we think about keeping these dep references up to date?

Reply:
I think following semantic versioning best practices with our npm package releases will be the most useful thing here. Breaking changes in npm packages should be major version bumps. So the project templates will not pick those up until someone manually comes in here, bumps the major version, and then makes any necessary compatibility changes.

The deps should be set up so that minor changes and patch changes get picked up for free (i.e. `^1.0.0` when we're ready for that)

Reply:
One thing that would probably also help would be to keep all of our HubSpot-official templates in a single repo so there is only one place that needs to be updated. Thinking about how this would scale, we could end up with tons of project template repos if each team creates their own.

Ultimately we will all probably have to be a little diligent about making sure that we're updating the templates whenever we make major changes to the npm packages. Longer term I'd like to hand ownerhsip of the boilerplate templates and repo to another team (maybe docs + onboarding?), and then they can be in charge of keeping things up to date and reviewing + approving PR's. I assume that some of the changes we make to our project templates in here would also require updates to either the docs or the getting started flow. So that's another +1 for the docs + onboarding team to take ownership of it